modules/distillery.cpp

```diff
@@ -1294,7 +1296,7 @@ void LapH::distillery::read_eigenvectors(){
 
   // running over all timeslices on this process, spread over a user-specified number
   // of threads
-  #pragma omp parallel for num_threads( param.evec_read_omp_num_threads )
+  #pragma omp parallel for num_threads( param.evec_read_omp_num_threads ) schedule(dynamic)
```

Comment:
why is dynamic better here?

Reply:
When the schedule is static, all threads are supposed get the same amount of work. That does not work out if the number of threads does not divide the number of eigenvectors cleanly and you end up waiting for (a) thread(s) which was given more work than the others. Since the reading takes a significant amount of time, the overhead from using dynamic scheduling is relatively small and the benefit of being independent of the divisibility outweighs it IMHO.